Backend/cat.js

```diff
@@ -1,20 +1,25 @@
 import words from './generic-food.js'
 
 // Ordered by rarity. Least rare first.
-const CAT_EYES = ["orange", "green", "blue", "gold"];
+const CAT_EYES_LEFT = ["orangeLeft", "greenLeft", "blueLeft", "goldLeft"];
+const CAT_EYES_RIGHT = ["orangeRight", "greenRight", "blueRight", "goldRight"];
 const CAT_COLORS = ["orange", "brown", "white", "tabby", "tshell", "calico", "black"];
-const CAT_HATS = ["partyBlue", "bowler", "cowboy", "partyPurple", "wizardBlue", "goggles", "cowboyPink", "wizardPurple", "partyEarth", "crown"];
+const CAT_HATS = ["partyBlue", "bowler", "brownHair", "cowboy", "partyPurple", "blondeHair", "wizardBlue", "goggles", "cowboyPink", "gingerHair", "wizardPurple", "partyEarth", "rainbow", "crown"];
 
 export default class Cat {
   // Generate a random cat.
   // `rarityGenerator` is a function that takes in a maximum and returns a number from 0 to that maximum,
   // according to a random distribution depending on the rarity the box should have.
   constructor(rarityGenerator) {
-    const eyeIndex = rarityGenerator(CAT_EYES.length - 1);
+    let eyeIndex = rarityGenerator(CAT_EYES_LEFT.length - 1);
     const colorIndex = rarityGenerator(CAT_COLORS.length - 1);
     const hatIndex = rarityGenerator(CAT_HATS.length - 1);
 
-    this.eyes = CAT_EYES[eyeIndex];
+    this.leftEye = CAT_EYES_LEFT[eyeIndex];
+    if (Math.random()>0.995) {
+      eyeIndex = rarityGenerator(CAT_EYES_RIGHT.length - 1);
```

Comment:
Maybe one way you could handle the rarity is to add both the left and right indices to the rarity?